docs/pages/guides/prebuild.md

```diff
@@ -0,0 +1,168 @@
+
+**With Prebuild** library authors, who know how configure their library better than anyone, can create a testable and versioned script (Expo Config Plugin) to automate adding the required configuration side-effects. This means library side-effects can be more expressive, powerful, and stable. For native code side-effects, we also provide: [AppDelegate Subscribers](/modules/appdelegate-subscribers) and [Android Lifecycle Listeners](/modules/android-lifecycle-listeners) which come standard in the default [Prebuild template](#templates).
+
+### Orphaned code
+
+When you uninstall a package you have to be certain you removed all of the side-effects required to make that package work. If you miss anything, it leads to orphaned code that you cannot trace back to any particular package, this code builds up and makes your project harder to understand and maintain.
+
+**With Prebuild** the only side-effect is the Config Plugin entry in a project's Expo config (`app.json`), which will fail to evaluate when the corresponding package has been uninstalled, meaning it's nearly impossible to create orphaned configuration.
+
+## Anti Pitch
+
+Here are some reasons _Expo Prebuilding_ might **not** be the right fit for a particular project. There are also some notes on how we plan on alleviating the existing drawbacks eventually.
+
+### React Native versioning
+
+`npx expo prebuild` generates native code based on the version of `expo` a project has installed, so a project with SDK 46 (`expo@46.0.0`) would generate a `react-native@0.69.4` app.
+
+Expo has a quarterly release cycle, whereas `react-native` releases major versions at random intervals, with stability patches following a few weeks after. This means there are times where you cannot use `npx expo prebuild` with the latest release of React Native. This could potentially be circumvented by using a custom [Prebuild template](#templates), but this is not recommended.
+
+If React Native moves to a more predictable and stable release schedule then we could potentially add support during a beta release period.
+
+### Platform compatibility
+
+Prebuild can only be used for native platforms that are supported by the Expo SDK. This means iOS and Android for the time being. With the exception of web, which doesn't require `npx expo prebuild` since it uses the browser instead of a custom native runtime.
+
+We plan to support more platforms in the future, but team size and limited resources prevent us from being able to maintain them at the moment. The stability of out-of-tree platforms like `react-native-macos` and `react-native-windows` are also considered.
```

Comment:
```suggestion
In the future, we would like to support additional platforms, such as `react-native-macos` and `react-native-windows`, but this is not currently a priority.
```

Reply:
I don't think we have any plans to support macos given the existence of Catalyst.